src/pages/ForgotPassword.js

```diff
@@ -0,0 +1,30 @@
+import "../styles/auth.css";
+import {NavMenu} from "../components/NavMenu";
+import {Link} from "react-router-dom";
+
+const ForgotPassword = () => {
+    return (
+        <>
+        <NavMenu />
+        <section className="form-ctn br-md">
+        <form>
+            <h2 className="text-center mg-bottom-md">Forgot Password</h2>
+            <div className="form-control">
+                <label htmlFor="email" className="fw-bold">Email address</label>
+                <input type="text" name="email" placeholder="name@gmail.com" />
```

Comment:
- here you forgot to add id attribute.
- it is a good practice to keep label tag's ```htmlFor``` attribute value and input tag's ```id``` attribute value same as it comes with inbuilt functionality i.e clicking on the label will focus its associated input element.

```suggestion
                <input type="text" name="email" id="email" placeholder="name@gmail.com" />
```